docs/README.md

```diff
@@ -52,3 +52,13 @@ likely caused by broken metadata needed to protect clients from cross-site-scrip
 style attacks.  Please [notify a maintainer](https://github.com/containers/podman#communications)
 so they may investigate how/why the `swagger.yaml` file's CORS-metadata is
 incorrect, or the file isn't accessible for some other reason.
+
+## Local Testing
+
+Assuming that you have the [dependencies](https://podman.io/getting-started/installation#build-and-run-dependencies)
```

Comment:
The dependencies listed there doesn't really matter. You need to install `python3-sphinx` and `python3-recommonmark`, at least that's how the packages are called on fedora, and also `pip install sphinx-markdown-tables`

Reply:
Thanks @Luap99 .  I kept the dependencies in, probably overdone, but included the python steps too.